dev/flipbook.tsx

```diff
@@ -267,7 +267,9 @@ function GradableRenderer(props: QuestionRendererProps) {
     const rendererRef = useRef<Renderer>(null);
     const [score, setScore] = useState<PerseusScore>();
 
-    useTimeout(() => setScore(undefined), 2500, !!score);
+    const clearScoreTimeout = useTimeout(() => setScore(undefined), 2500, {
+        schedulePolicy: SchedulePolicy.OnDemand,
```

Comment:
🤔  I noticed this too where I used `useTimeout`... it would have been nice for it to start automatically.